PyQt5-stubs/QtCore.pyi

```diff
@@ -2376,6 +2376,20 @@ class Qt(sip.simplewrapper):
     WindowActive = ...  # type: Qt.WindowState
 
     class WindowType(int):
+        def __init__(self, f: typing.Union['Qt.WindowType', int]) -> None: ...
+        def __int__(self) -> int: ...
+        def __invert__(self) -> 'Qt.WindowFlags': ...
```

Comment:
Actually, this is wrong. I need to ditch away all the `WindowType `modifications because they break the Liskov principle : if you inherit `int`, overloaded method should return something which is `int `compatible, so clearly` Qt.WindowFlags `  which does not inherit `int `is not a good candidate. This applies to all the methods below.

I guess we'll have to do just like in C++, where `WindowType `operations are not correctly checked because they become `int `after an operation.

Reply:
That's why I initially ditched the `int `inheritance.